templates/_helpers.yaml

```diff
@@ -70,12 +76,30 @@
         name: {{ $config.secretName }}
         key: {{ default "value" $config.secretKey }}
   {{- end }}
+{{- end }}
+
+{{- define "custom_airflow_environment_config" }}
+  # Dynamically created environment variables
+  {{- range $i, $config := .Values.env }}
+  - name: {{ $config.name }}
+    value: {{ $config.value | quote }}
+    {{- if eq $.Values.executor "KubernetesExecutor" }}
+    - name: AIRFLOW__KUBERNETES_ENVIRONMENT_VARIABLES__{{ $config.name }}
+      value: {{ $config.value | quote }}
```

Comment:
@kushalmalani should these be un-indented to line up with the ones right above it?

Reply:
oh yeah! Copy pasting in VS Yaml is horrible. Good catch! I will get this in and fix the test cases